tests/1-login.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
+const user = {
+  name: 'Jane Doe',
+  email: 'customer@practicesoftwaretesting.com',
+  password: 'welcome01'
+}
+
+test('login', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
+  await page.locator('#email').fill(user.email);
+  await page.locator('#password').fill(user.password);
+  await page.locator('input.btnSubmit').click();
+  await expect(page).toHaveURL('https://practicesoftwaretesting.com/account');
```

Comment:
If you add baseUrl, then this expression can be simplified using RegExp

```suggestion
  await expect(page).toHaveURL(/\/account$/);
```